test/FunctionalTests/Microsoft.OData.Core.Tests/UriParser/Metadata/ODataUriResolverTests.cs

```diff
@@ -78,6 +78,14 @@ private static void VerifyEnumVsStringFilterExpression(FilterClause filter)
             bin.Right.ShouldBeEnumNode(enumtypeRef.EnumDefinition(), "2");
         }
 
+        private static void VerifyEnumVsStringFilterExpressionforHas(FilterClause filter)

```

Comment:
If this method is only used at one place, maybe remove this function and add the codes into the test directly?